### ISSUE TITLE
fscrypt fails to build on 32bit x86: mismatched types int32 and int64

### ISSUE THREAD

**Issue by paride** [google/fscrypt#233]
fscrypt v0.2.8 fails to build on i386 with the following error:
```
src/github.com/google/fscrypt/cmd/fscrypt/errors.go:121:19: invalid operation: statfs.Bsize != pagesize (mismatched types int32 and int64)
```
As i386 is a Debian release architecture this bug prevents the migration of the newer fscrypt versions to Debian testing.

**Comment by ebiggers** [comment]
Thanks for reporting this; sorry for the trouble.  I should have tried to get a 32-bit build target in Travis CI the last time we ran into an issue like this.  Trying it now...

**Comment by ebiggers** [comment]
Fixed by https://github.com/google/fscrypt/pull/235/commits/3b075f2fda880256b3d9de2e6197a224adc0a39f.  Also, https://github.com/google/fscrypt/commit/3c15a7a05bd7673d2ca4650dd1525df408d20728 added a 32-bit build to Travis CI, so this shouldn't happen again.

@josephlr, it might be necessary to release v0.2.9 already, to get this fix out.  Otherwise some people still can't upgrade from v0.2.5, unfortunately!

In the mean time, @paride, please feel free to carry the fix in the Debian package for v0.2.8.

**Comment by ebiggers** [comment]
I've released v0.2.9 which includes this fix.
